src/shims/windows/unistd.cc

```diff
@@ -0,0 +1,28 @@
+#include <unistd.h>
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN 1
+#endif
+#include <Windows.h>
+
+int kill(pid_t pid, int sig) {
```

Comment:
Nice! Can you please add a header comment for the function though?

Reply:
I already have actually! See `unistd.h`: https://github.com/ray-project/ray/pull/6362/files/0e244e3c9e8a1b47d42f74b302b2ee788640254b#diff-a133f668cbcc8d36673c5b1b0e9c79b3R8

Reply:
Ah I see. What do `__declspec(` and `deprecated` do?

Reply:
It's basically the equivalent of GCC's `__attribute__((deprecated(msg)))`, but for Microsoft compilers -- they emit warning that message as a warning the function is used.

Reply:
Whoops sorry for the typo, edited my comment.